library/x509_crt.c

```diff
@@ -2990,6 +2991,137 @@ static int x509_crt_verify_chain(
     }
 }
 
+#ifdef _WIN32
+/* ??? */
```

Comment:
I'm not sure what this comment is about

Reply:
I plan to fill in the headers needed for _WIN32.

Reply:
:rocket:

I can not see into private mbedtls CI internal tests.  If they require changes to link with -lws2_32, then please make the appropriate changes, or else suggest that I remove _WIN32 support here and instead fall back on _WIN32 to using the local inet_pton() implementation that I provided.